### PR TITLE
CI: Add nightly prerelease builds

### PR DESCRIPTION
**Pull request by guicaulada**

### CODE REVIEW

File: scripts/drone/rgm.star [hunk 1]
```diff
@@ -73,6 +73,25 @@ tag_trigger = {
     },
 }
 
+nightly_trigger = {
+    "event": {
+        "include": [
```
Comment (zerok) [inline scripts/drone/rgm.star:78]: Do you have to use `include` here or could you just do something like `"event": ["promote", "cron"],`? Not sure what the difference here is 🙂

Comment (guicaulada) [reply scripts/drone/rgm.star:67]: Tried it using `"event": ["promote", "cron"]` didn't seem to work, maybe I can't have 2 events on the same trigger? It works now. But you're correct, theres no difference https://docs.drone.io/pipeline/triggers/

File: public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue [hunk 2]
```diff
@@ -36,7 +36,7 @@ composableKinds: DataQuery: {
 				// Allows to group the results.
 				groupBy: [...string]
 				// Sets the maximum number of nodes in the flamegraph.
-				maxNodes?:        int64
+				maxNodes?:           int64
```
Comment (aocenas) [inline public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue:39]: Seems like this is why our squad is a reviewer here, at the same time I assume this is not intentional :)

Comment (guicaulada) [reply public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue:39]: It's not! Not sure how that got there, it was probably a hook?

Comment (guicaulada) [reply public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue:39]: Trying to rever the change on 42ad201 resulted in an empty commit, it's caused by a pre-commit hook.

File: public/locales/pseudo-LOCALE/grafana.json
Comment (joshhunt) [inline public/locales/pseudo-LOCALE/grafana.json:1]: if you like you can merge/rebase main to your PR to remove this from the diff.

Comment (guicaulada) [reply public/locales/pseudo-LOCALE/grafana.json:1]: Done!

**Review by kminehart — APPROVED** [review]

**Pull request merged**